consensus/module_consensus_debugging.go

```diff
@@ -80,3 +67,11 @@ func (m *consensusModule) GetLeaderForView(height, round uint64, step uint8) uin
 	}
 	return uint64(leaderId)
 }
+
+func (m *consensusModule) PushStateSyncMetadataResponse(minHeight, maxHeight uint64) {
+	m.metadataReceived <- &typesCons.StateSyncMetadataResponse{
+		PeerAddress: "",
```

Comment:
This should accept the whole response as input, not jus part of the data.

Think: `What will we need in the future, and is this what I need or is it a hack? How much more effort is it to do it properly? If the effort is really small, let's do it properly`

Reply:
Couldn't have said it better myself ;)